src/components/variant/Variant.tsx

```diff
@@ -1,37 +1,93 @@
-import { Card, Grid, Typography } from '@material-ui/core';
+import { Card, CardContent, Grid, Typography } from '@material-ui/core';
+import { UIBuild } from 'evergreen.js/lib/models';
 import * as React from 'react';
 import '../../styles.css';
 
+interface State {
+  name: string,
+  statusCount: { [id: string]: number }
+}
+
 class Props {
-  public name: string;
+  public build: UIBuild;
 }
 
-export class Variant extends React.Component<Props> {
+export class Variant extends React.Component<Props, State> {
+
   constructor(props: Props) {
     super(props);
+    const statusCount = {};
+    for (const task of this.props.build.Build.tasks) {
+      if (statusCount[task.status]) {
+        statusCount[task.status] = statusCount[task.status] + 1;
```

Comment:
nit: I would use ++

Reply:
huh, I didn't know that was a thing you could do in Javascript, I assumed it wouldn't actually update anything in the object. fixed